db/migrate/20210129033100_add_timestamps_to_crawl_receipts.rb

```diff
@@ -0,0 +1,5 @@
+class AddTimestampsToCrawlReceipts < ActiveRecord::Migration[6.0]
+  def change
+    change_table(:crawl_receipts) { |t| t.timestamps}
```

Comment:
huh, didn't know `timestamps` was defined like this, cool!

Reply:
Had to wait for the timecop gem pr to land before being able to fix the unit tests! Fixing them now 👍🏼